srcpkgs/dmd-bootstrap/template

```diff
@@ -56,9 +56,9 @@ do_install() {
 	vbin dmd
 
 	vmkdir etc
-	cat > $DESTDIR/etc/dmd.conf <<EOF
-[Environment]
-DFLAGS=-I/usr/include/dlang/dmd -L-L/usr/lib${XBPS_TARGET_WRODSIZE} -L--export-dynamic
+	cat > $DESTDIR/etc/dmd.conf <<- EOF
+	[Environment]
+	DFLAGS=-I/usr/include/dlang/dmd -L-L/usr/lib${XBPS_TARGET_WRODSIZE} -L--export-dynamic
 EOF
```

Comment:
If you're going to indent the block, might as well indent the EOF too.

Reply:
You, with the good ideas. :P